src/Workspaces/CoreTest/SolutionTests/SolutionTests.cs

```diff
@@ -1242,76 +1242,27 @@ public TestLanguageServiceB()
             }
         }
 
-        [Fact]
-        public void TestDocumentFileAccessFailureMissingFile()
-        {
-            var solution = new AdhocWorkspace().CurrentSolution;
-
-            WorkspaceDiagnostic diagnostic = null;
-
-            solution.Workspace.WorkspaceFailed += (sender, args) =>
-            {
-                diagnostic = args.Diagnostic;
-            };
```

Comment:
❓ If we kept this eventing for compatibility, why is the test removed?

Reply:
I'll add a test for the event.